Lib/enum.py

```diff
@@ -731,14 +731,16 @@ def __contains__(cls, value):
         """
         if isinstance(value, cls):
             return True
-        try:
-            cls(value)
-            return True
-        except ValueError:
-            return (
-                    value in cls._unhashable_values_    # both structures are lists
-                    or value in cls._hashable_values_
-                    )
+        if issubclass(cls, Flag):
```

Comment:
Is it possible for this code to be executed in the future when `Flag` is None? I think we do some `if Flag is not None` in other places.